### DISCUSSION TITLE
Issue on WSL with Localhost

### DISCUSSION THREAD

**Discussion by somecallmeash** [oobabooga/text-generation-webui#5235] [Q&A]
I'm not sure if this is a WSL issue or something gone wrong with my install of the library, but after getting the cuda tools installed in my WSL environment I installed this package and used the downloading script to get a vicuna model to try out, but when I run the server I get 

```
ValueError: When localhost is not accessible, a shareable link must be created. Please set share=True or check your
proxy settings to allow access to localhost.
```

I can't find much about this and looking around for issues with WSL and localhost had a few ideas about the firewall (already disabled) and using netsh to map the correct port in powershell, so I did that with port 7860, but I am still getting the same error. I also tried shutting down and restarting WSL because a serverfault question said the WSL feature that exposes localhost in windows can sometimes break.

Has anyone encountered this and hopefully solved it?

**Comment by DLCIncluded** [comment]
Having the same issue, but on just the basic start_windows.bat 
Havent been able to solve, ran as admin, deleted, and tried an older version, still no luck...

**Reply by DLCIncluded** [reply]
Actually, i just got past this - spoke too soon try this:
in the base folder you should have a CMD_FLAGS.txt file
add --share to the end
save and run the bat file again, i was able to get it to launch and get into the UI
not sure if this fixes everything but its a step in the right direction

edit: i dont love this option as it opens it to public access with gradio it seems
so make sure to add `--gradio-auth username:password`
